scripts/executeProposal.ts

```diff
@@ -15,12 +15,12 @@ async function main() {
   const { address: governanceAddress } = await hre.deployments.get("GovernorBravoDelegator");
   const governance = GovernorBravoDelegate__factory.connect(governanceAddress, signer);
 
-  const tx = await governance.execute(1,
+  const tx = await governance.execute(9,
```

Comment:
Guessing for testing you were just manually changing the proposal IDs and thats where the 9 comes from?

Reply:
Yes!